workers/unity/Packages/io.improbable.gdk.tools/GdkToolsConfigurationProvider.cs

```diff
@@ -221,16 +237,32 @@ private void DrawCodeGenerationOptions()
                 }
             }
 
+            EditorGUIUtility.labelWidth = previousWidth;
+        }
+
+        private void DrawDevAuthTokenOptions()
+        {
+            var previousWidth = EditorGUIUtility.labelWidth;
+            EditorGUIUtility.labelWidth = 180;
+
             GUILayout.Label(DevAuthTokenSectionLabel, EditorStyles.boldLabel);
             using (new EditorGUI.IndentLevelScope())
             {
                 toolsConfig.DevAuthTokenLifetimeDays =
-                    EditorGUILayout.IntSlider(DevAuthTokenLifetimeLabel, toolsConfig.DevAuthTokenLifetimeDays, 1, 90);
-
-                toolsConfig.SaveDevAuthTokenToFile = EditorGUILayout.Toggle("Save token to file", toolsConfig.SaveDevAuthTokenToFile);
+                    EditorGUILayout.IntSlider(new GUIContent(DevAuthTokenLifetimeLabel,
+                            "Sets the lifetime for requested development authentication tokens, between 1 and 90 days. " +
+                            "Changes made to this setting do not affect already requested tokens."),
+                        toolsConfig.DevAuthTokenLifetimeDays, 1, 90);
+
+                toolsConfig.SaveDevAuthTokenToFile = EditorGUILayout.Toggle(new GUIContent("Save token to file",
+                        "Sets whether to save the development authentication to a file, instead of in player preferences."),
```

Comment:
```suggestion
                        "Sets whether to save the development authentication token to a file, instead of in player preferences."),
```